src/approaches/interactive_learning_approach.py

```diff
@@ -265,6 +265,8 @@ def _query_policy(s: State) -> Optional[GroundAtomsHoldQuery]:
                    (not strict and score >= self._best_score):
                     atoms_to_query.add(atom)
                     self._best_score = score
+            if not atoms_to_query:
```

Comment:
interesting, why this change? will this have the effect that the episode terminates as soon as there are no more atoms to query? but wouldn't that not be what we want, if for example, some tantalizing atoms are waiting for us later on in the episode?

Reply:
ohhh I forgot that's what the `None` value means, I was annoyed b/c it made my videos ugly but I'll change it back lmao

Reply:
is the ugliness because of the empty set? feel free to special case that in the caption generation